snippets/tf/snippets.tf

```diff
@@ -0,0 +1,44 @@
+provider "aws" {
+  region = "${var.region}"
+}
+
+##### Buckets
+
+module "bucket-stage" {
+  source       = "../tf/storage"
+  environment  = "stage"
+  region       = "${var.region}"
+  region_short = "${var.region_short}"
+}
+
+module "bucket-prod" {
+  source       = "../tf/storage"
+  environment  = "prod"
+  region       = "${var.region}"
+  region_short = "${var.region_short}"
+}
+
+module "prod-alerts" {
+  source = "../tf/alerting"
+  region = "${var.region}"
+  fqdn   = "${var.fqdn_prod}"
+  name   = "${var.alarm_name_prod}"
+}
+
+# do we want stage alerts?
```

Comment:
@jgmize and I discussed this yesterday, we can have the alarms, but they won't raise any emails/IRC notifications.